crates/ext-processor/src/headers.rs

```diff
@@ -43,36 +50,48 @@ mod tests {
     #[test]
     fn test_serialize_decision_sfv() -> Result<(), Box<dyn std::error::Error>> {
         assert_eq!(
-            serialize_decision_sfv(Decision {
-                accept: 0.0,
-                restrict: 0.0,
-                unknown: 1.0
-            })?,
-            "accept=0.0, restrict=0.0, unknown=1.0"
+            serialize_decision_sfv(
+                Decision {
+                    accept: 0.0,
+                    restrict: 0.0,
+                    unknown: 1.0
+                },
+                Outcome::Suspected
+            )?,
+            "accept=0.0, restrict=0.0, unknown=1.0, score=0.5, outcome=\"suspected\""
         );
         assert_eq!(
-            serialize_decision_sfv(Decision {
-                accept: 0.0,
-                restrict: 1.0,
-                unknown: 0.0
-            })?,
-            "accept=0.0, restrict=1.0, unknown=0.0"
+            serialize_decision_sfv(
+                Decision {
+                    accept: 0.0,
+                    restrict: 1.0,
+                    unknown: 0.0
+                },
+                Outcome::Restricted
+            )?,
+            "accept=0.0, restrict=1.0, unknown=0.0, score=1.0, outcome=\"restricted\""
         );
         assert_eq!(
-            serialize_decision_sfv(Decision {
-                accept: 1.0,
-                restrict: 0.0,
-                unknown: 0.0
-            })?,
-            "accept=1.0, restrict=0.0, unknown=0.0"
+            serialize_decision_sfv(
+                Decision {
+                    accept: 1.0,
+                    restrict: 0.0,
+                    unknown: 0.0
+                },
+                Outcome::Accepted
```

Comment:
I didn't notice this before, but this should probably be marked `Outcome::Trusted` since it's a 1.0 accept value.